bin/fuel-convert

```diff
@@ -10,6 +10,18 @@ from fuel.converters.base import MissingInputFiles
 from fuel.datasets import H5PYDataset
 
 
+class SplitOutputAction(argparse.Action):
+    def __call__(self, parser, namespace, values, option_string=None):
+        absolute_path = os.path.abspath(values)
+        if os.path.isdir(absolute_path):
```

Comment:
What I had in mind was something similar to `cp`'s interface: if you pass it a directory as the second argument it'll re-use the filename in the first argument and copy *inside* that directory.

If you pass a directory as the output path, `fuel-convert` will use the default file name provided by the converter. Otherwise, it'll use the user-provided filename.

@bartvm you're right that if the user overwrites the filename, the corresponding `Dataset` class won't be able to find the file. I must admit that I can't think of a good use case for choosing a filename other than the default one. If nobody else can find a use case for that, I'd be happy to simplify the interface and just allow the user to specify the output directory.